initial commit

AleksandrPok opened the pull request.

AleksandrPok committed:
initial commit

vicire commented on pom.xml:
```diff
@@ -23,6 +23,22 @@
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.5.2</version>
+        </dependency>
```
You need one of these two dependecies

vicire commented on src/test/java/CustomExecutorTest.java:
```diff
@@ -0,0 +1,24 @@
+        Integer actual = customExecutor.getSum();
+        assertEquals(expected, actual, "Expected: " + expected + "Actual: " + actual);
+    }
+}
```
add empty line

vicire commented on src/test/java/CustomForkJoinTest.java:
```diff
@@ -0,0 +1,24 @@
+        Integer actual = customForkJoin.compute();
+        assertEquals(expected, actual, "Expected: " + expected + "Actual: " + actual);
+    }
+}
```
same here

vicire commented on src/test/java/CustomForkJoinTest.java:
```diff
@@ -0,0 +1,24 @@
+
+    @Test
+    public void compute_Ok() {
+        Integer expected = NumbersProducer.getIntegerList().stream()
+                .reduce(Integer::sum).get();
```
You can put this logic in setUp

vicire commented on src/test/java/CustomExecutorTest.java:
```diff
@@ -0,0 +1,24 @@
+
+    @Test
+    public void getSum_Ok() {
+        Integer expected = NumbersProducer.getIntegerList().stream()
+                .reduce(Integer::sum).get();
```
Same here

AleksandrPok committed:
removed unnecessary dependency;
added empty lines in CustomExecutorTest & CustomForkJoinTest;
removed expected initialisation to setUp()
